README.md

```diff
@@ -1,2 +1,29 @@
-# Samples
-Example pointcloud data that can be used for example displays and tests.
+# Testing Data (for point cloud visualization)
+
+Example point cloud data that can be used for example displays and tests. 
+
+**NOTE:** Creating these datasets requires [PDAL](https://pdal.io). If you don't have pdal installed, please checkout their [installation guides]((https://pdal.io/download.html)) or use via [docker](https://pdal.io/quickstart.html).
+
+## Data description
+As of now, there are three sets of point cloud data that are available for testing. 1. **ASPRS LAS specified data** (data in little-endian format) that can be found in the `las` directory. 2. **Entwine Point Format** (EPT) data that can be found in the `ept/NorthCarolina` directory. And 3. A collection of random point clouds that can be found in the `random` directory.
+
+### las
+A few notes on the naming convention of generated files:
+
+- Files are named with the following convention:`v<LAS minor version number>d<LAS data format>.las`. Therefore, `v1d1.las` is a point cloud with a minor version of `1` and a data format of `1`.
```

Comment:
I was wondering how to maintain a litany of versions of a format. Proper file-naming indeed seems like a good way to go 👍

Reply:
thanks